tools/setup_perf_cluster.sh

```diff
@@ -143,7 +143,7 @@ function install_non_istio_svc() {
  Execute kubectl -n $FORTIO_NAMESPACE run fortio1 --image=istio/fortio --port=8080
  Execute kubectl -n $FORTIO_NAMESPACE expose deployment fortio1 --target-port=8080 --type=LoadBalancer
  Execute kubectl -n $FORTIO_NAMESPACE run fortio2 --image=istio/fortio --port=8080
- Execute kubectl -n $FORTIO_NAMESPACE expose deployment fortio2 --target-port=8080
+ Execute kubectl -n $FORTIO_NAMESPACE expose deployment fortio2 --target-port=8080 --type=LoadBalancer
```

Comment:
I'm trying to remember why I didn't put both as load balancer, there was a reason (maybe to compare perf/setup) but I forgot :/
technically for the fortio reference we don't really need direct access to the second service as it can be proxied by the first one

Reply:
The gce ingress controller [requires](https://github.com/kubernetes/ingress-gce#overview) a service to be exposed using type `LoadBalancer` or type `NodePort`. This is why you could not get the fortio2 ingress to work.

Reply:
not sure we need to expose fortio2 through ingress, it's going to consume another external ip

Reply:
If we want to create an ingress for fortio2, then we need the fortio2 service to be of type `LoadBalancer` or type `NodePort`. This is a requirement of the gce ingress controller. I saw the note `# Doesn't work somehow...` in the script and came up with this fix.